src/atoms/connectWalletButton/connectWalletButton.styles.ts

```diff
@@ -0,0 +1,36 @@
+import { Styled } from '../../theme';
+import { ConnectButtonProps } from './types';
+
+export const ConnectButton = Styled.button<ConnectButtonProps>`
+  svg {
+    cursor: pointer;
+    fill: ${(props) => props.theme.textShades.SHADE_MINUS_3}!important;
```

Comment:
Do you still need the `!important` here? Better to avoid

Reply:
Thanks missed this